client/modules/App/components/Overlay.jsx

```diff
@@ -30,25 +59,46 @@ class Overlay extends React.Component {
       children
     } = this.props;
     return (
-      <div className="overlay">
-        <div className="overlay__content">
-          <section
-            tabIndex="0"
-            role="main"
-            aria-label={ariaLabel}
-            ref={(element) => { this.overlay = element; }}
-            className="overlay__body"
-          >
-            <header className="overlay__header">
-              <h2 className="overlay__title">{title}</h2>
-              <button className="overlay__close-button" onClick={this.close}>
-                <InlineSVG src={exitUrl} alt="close overlay" />
-              </button>
-            </header>
-            {children}
-          </section>
+      <a tabIndex={-42} onKeyDown={this.keyPressHandle} id="overlay__anchor">
+        <div className="overlay">
+          <div className="overlay__content">
+            <section
+              tabIndex="0"
+              role="main"
+              aria-label={ariaLabel}
+              ref={(node) => { this.node = node; }}
+              className="overlay__body"
+            >
+              <header className="overlay__header">
+                <h2 className="overlay__title">{title}</h2>
+                <button className="overlay__close-button" onClick={this.close} >
+                  <InlineSVG src={exitUrl} alt="close overlay" />
+                </button>
+              </header>
+              {children}
+            </section>
+          </div>
         </div>
-      </div>
+      </a>
+    // <div className="overlay">
```

Comment:
rather than just commenting out this code, could you remove it? if we want to look at the old version we can just revert this file to a previous commit ✨

Reply:
ok, forgot about it :sweat_smile: